src/directives/for.js

```diff
@@ -1,10 +1,13 @@
 import { transitionIn, transitionOut, getXAttrs } from '../utils'
 
-export function handleForDirective(component, el, expression, initialUpdate) {
+export function handleForDirective(component, el, expression, initialUpdate, extraVars) {
     if (el.tagName.toLowerCase() !== 'template') console.warn('Alpine: [x-for] directive should only be added to <template> tags.')
 
     const { single, bunch, iterator1, iterator2 } = parseFor(expression)
 
+    // Get the depth level for the current iteration
```

Comment:
required when we use
```html
<template x-for="...">
    <template x-for="...">
         ...
    </template>
</template>
```
because elements from different loops end on the same DOM level and we need to differentiate apart.